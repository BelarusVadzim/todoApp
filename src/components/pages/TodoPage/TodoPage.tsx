import React from 'react';
import { TodoTemplate } from 'components/templates';

const TodoPage: React.FC = () => <TodoTemplate />;

export default TodoPage;
