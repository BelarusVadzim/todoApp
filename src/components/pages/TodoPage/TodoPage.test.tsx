import { render } from '@testing-library/react';
import TodoPage from './TodoPage';

jest.mock('components/templates', () => ({
  TodoTemplate: jest.fn(() => <div>TodoTemplate</div>),
}));

describe('<TodoPage />', () => {
  it('should render properly', () => {
    const component = render(<TodoPage />);

    expect(component.asFragment()).toMatchSnapshot();
  });
});
