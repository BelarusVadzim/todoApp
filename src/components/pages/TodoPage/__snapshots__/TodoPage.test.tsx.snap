// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TodoPage /> should render properly 1`] = `
<DocumentFragment>
  <div>
    TodoTemplate
  </div>
</DocumentFragment>
`;
